data_dir/nemesis.yml

```diff
@@ -16,7 +16,6 @@
   - run_with_gemini = False
   - limited = True
   - topology_changes = True
-  - tablets_phase_1_supported = False
```

Comment:
What does it mean for nemesis that aren't supported with "Tablets"?
Will they fail?
In this case, it can't go like that and either each nemesis will have a special handling for that, or you get and approval from whoever is to approve it that we won't run all these nemesis anymore and we keep this filter but set it to be the default for all longevities.

Reply:
@roydahan he's removing this label that @ShlomiBalalis introduced.

@yarongilor don't really know which one works and which one doesn't 

I'm o.k. with the assumption they are all working, we didn't limited it in any case, just in few shlomo was testing prior to making tablets default.

it's water under the bridge now

Reply:
@roydahan , as @fruch mentioned, it is already "too late" (after tablets are enabled by default).
See, for example - https://github.com/scylladb/scylla-cluster-tests/issues/7671

We do roughly know what isn't supported with tablets: CDC, LWT, resharding and few more.
So there are few fixes to add. and there are few "feature specific" longevities to adjust to temporarily run without tablets.